dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/impl/service/identifiable/IdentifiableServiceImpl.java

```diff
@@ -247,10 +248,10 @@ public I save(I identifiable) throws IdentifiableServiceException {
       try {
         urlAliasService.validate(identifiable.getLocalizedUrlAliases());
       } catch (Exception e) {
-        throw new IdentifiableServiceException("Validation error: " + e);
+        throw new ValidationException("Validation error: " + e, e);
       }
 
-      if (savedIdentifiable.getLocalizedUrlAliases() != null) {
+      if (!savedIdentifiable.getLocalizedUrlAliases().isEmpty()) {
```

Comment:
Ist an dieser Stelle sichergestellt, dass es nicht `null` sein kann? Sollte es leer sein, wird die for-Schleife einfach "übersprungen" und nichts weiter passiert.

Reply:
ist gefixed.